protos/lightstreamer.proto

```diff
@@ -10,7 +10,7 @@ message ServerInfo {
  string nodeVersion = 4;
  string nodeStatus = 5;
  uint64 blockHeight = 6;
```

Comment:
Should the blockHeight and blockHash be optional field instead of required? In the case when the node status is stopped or errored, the blockHeight and blockHash are not available. I saw those fields have the default values of 0 and empty string when not available, alternatively they can be omitted if they are not available.

Reply:
All values in proto3 are optional by default.

Reply:
There is major difference setting `optional` in proto3
https://protobuf.dev/programming-guides/proto3/#json
With `optional` field, it will support explicit presence.  https://github.com/protocolbuffers/protobuf/blob/main/docs/field_presence.md#presence-in-proto3-apis
The main difference for explicit presence is it will set values are always serialized, including default values. By default, its no presence, where default value will not be serialized. When you get the default value in no presence, there is no way to tell whether is intentionally set to default or it was never set.

Reply:
This is a good summary of the difference https://github.com/protocolbuffers/protobuf/blob/main/docs/field_presence.md#semantic-differences

Reply:
Cool ty makes more sense. Seems like you have a good understanding of optionality/protobuf reqs, do you want to make a new PR for this? I think it is out of scope for this change, but nothing is blocking it.

Reply:
It will affect the way the getServerInfo response is set. If the node status is stopped or errored, then the block height and block hash can be omitted instead of setting a default value.